### ISSUE TITLE
AVOID_USING_STRUCTNEW false positive with struct ordered

### ISSUE THREAD

**Issue by davidsf** [cflint/CFLint#607]
When declaring a ordered struct, I do:

```
a  = structNew( "ordered" );
```
and cflint complains about the use of structnew. I don't see in the docs another way to declare it.

**Comment by KamasamaK** [comment]
@ryaneberly That does not initialize it as an ordered struct. There is a literal syntax for ordered structs (see #534), but that cannot be used for initializing an empty ordered struct.

**Comment by ryaneberly** [comment]
Good point.  Maybe that rule in lint should not fire if StructNew is used
with parameters!

**Comment by KamasamaK** [comment]
So I just found that there actually is a way to declare an empty ordered struct using the literal syntax. It is:
`var orderedStruct = [:]`

However, it produces a parse error due to #534.
